readme.md

```diff
@@ -7,30 +7,28 @@
   <a href="https://discord.gg/3CVqX5bphG" alt="Discord"><img src="https://img.shields.io/discord/829309363213565982?label=Discord&style=flat&logo=discord&logoColor=white"></img></a>
   <a href="https://github.com/raluvy95/jolly/issues" alt="Issues"><img src="https://img.shields.io/github/issues/raluvy95/jolly"></img></a>
 </p>
-A discord bot designed for one server at once (private bot). Rewritten in
-TypeScript using Deno. The predecessor variant of this can be found in
+A discord bot designed for my server (private bot). Rewritten in
+TypeScript using Deno. Its predecessor can be found in
 <a href="https://github.com/raluvy95/maffin">another repository</a>
 
 You can contribute whatever you want to improve the bot with additional features
-and fixed bugs.
+bug fixes.
```

Comment:
``and bug fixes`` would have probably been better but this is just a minor remark and might just be an oopsie